net/sslsplit/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2016-2018 OpenWrt.org
```

Comment:
Don't assign copyright to OpenWrt.org unless you're affiliated, either omit or use your own.